scripts/reformat_copyright.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
When should this be called?
It looks like it's not used in the CI

Reply:
It's only a helper script for end users (that is us) to use manually when fixing up a PR.

> There is a helper script in `./scripts/reformat_copyright.sh` that can be used to update the copyright headers in new files in PRs.